.github/workflows/ci-workflow.yml

```diff
@@ -93,6 +94,17 @@ jobs:
           ${{ env.MAVEN_COMMAND }}
           ${{ env.MAVEN_PROFILES }}
           ${{ env.MAVEN_PROJECTS }}
+      - name: upload-on-failure
+        uses: actions/upload-artifact@v2
+        with:
+          name: surefire-reports-ubuntu-en
+          path: |
+            ./**/target/surefire-reports/*.txt
+            ./**/target/surefire-reports/*.xml
+          retention-days: 3
+        if: failure()
```

Comment:
Unfortunately it looks like this does not pull any files for the current failure on JDK 11, so perhaps capturing some other output may be helpful.

Reply:
This is actually good.  Now we know that there is no issue with the test itself, but rather with the maven invocation of surefire.